build.sh

```
@@ -1,46 +1,7 @@
-cd tmp
-ar -x ../flexibee_${VERSION}_all.deb
-cd ..
-#cd debian
-#tar xzvf ../tmp/control.tar.gz
-#cd ..
-mkdir data
-cd data
-tar xzvf ../tmp/data.tar.gz
-cd ..
-
 CHANGES=`git log -n 1 | tail -n+5`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)